_config.yml

```diff
@@ -34,6 +40,19 @@ defaults:
     values:
       layout: "tour"
       permalink: "/tour/:basename/"
+  -
+    scope:
+      path: ""
```

Comment:
Can you explain these changes?  I am not sure how they are used. Why is `path` empty?  What is `skiph1fortitle`?  I suspect this section is somehow for 1.2 doc specific vars, but not sure.

Reply:
Path shouldn't be empty.  Also skiph1fortitle can be removed.  This should be fixed now.